manifests/0000_70_dns-operator_00-cluster-role.yaml

```diff
@@ -72,6 +73,7 @@ rules:
   - get
   - list
   - watch
+  - update
```

Comment:
We really only need to add update permission for the `clusterroles` resource, right?  The more we can restrict access, the better.

Reply:
done